specs/v2-sketch.yaml

```diff
@@ -0,0 +1,365 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: CircleCI API v2
+  license:
+    name: MIT
+servers:
+  - url: http://circleci.com/api/v2
+paths:
+  /me:
+    get:
+      summary: Return basic user information of the owner of the requested token
+      operationId: me
+      responses:
+        '200':
+          description: A map of simple user information
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Me"
+  /workflow/{id}:
+    get:
+      summary: Return summary fields of a workflow by ID
+      operationId: GET workflow by ID
+      parameters:
+        - name: id
+          in: path
+          description: ID of the workflow
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: An object with workflow information
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Workflow"
+  /workflow/{id}/jobs:
+    get:
+      summary: Return all the known jobs of a given workflow
+      operationId: GET workflow jobs by ID
+      parameters:
+        - name: id
+          in: path
+          description: ID of the workflow
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - name: page_token
+          in: query
+          description: Page token
+          schema:
+            type: string
+        - name: page_size
+          in: query
+          description: Number of jobs to return per page of results, default 50
```

Comment:
Max?

Reply:
I'm working up another document to get some discussion on how pagination will really work. I'm currently leaning towards each API having a documented minimum and maximum and defaulting to the maximum value. Haven't run this by anybody yet so consider this WIP.

Reply:
I'm also considering not exposing `page_size` at all and using a fixed value instead. Adding `page_size` later is a non-breaking change but we need pagination in from the beginning.